substrate-node/doc/terminology.md

```diff
@@ -1,21 +0,0 @@
-# Tfchain terminolog
```

Comment:
I know most of this is deprecated  but the defintion of a Validator or Council member are nowhere else